public/scripts/randomPet.js

```diff
@@ -0,0 +1,33 @@
+(function(module) {
+  var randomPets = {};
+  randomPets.all = [];
+
+randomPets.getRandom = function() {
+  $.getJSON('https://api.petfinder.com/pet.getRandom?format=json&key=8dc33d8c70fd213dc0874e9deaa0a2fd&age=Senior&output=full&count=6&callback=?',function(data){
```

Comment:
Having your API key here is a no-no. You should find another way to handle that, such as putting it in another file that's then gitignored.